### DISCUSSION TITLE
Custom character counter with Mentions

### DISCUSSION THREAD

**Discussion by gezzasa** [ueberdosis/tiptap#3079] [General Questions]
Is there a way to set character amounts for certain mentions? I would like to use the character counter extension but we have custom mentions, each with their own character limit.

```javascript
[
  // char is added here to include the maximum character size of the dynamic tag
  { id: 1, label: 'tag1', char: 20 },
  { id: 2, label: 'tag2', char: 20 },
  { id: 3, label: 'tag3', char: 20 },
  { id: 4, label: 'tag4', char: 35 },
  { id: 5, label: 'tag5', char: 20 },
],
```

I'm not quite sure how to approach this issue.

**Comment by msichterman** [comment]
Also wondering what best practices are here
